src/publisher/index.ts

```diff
@@ -168,9 +169,9 @@ export class Publisher {
   setOptions(options = {} as PublishOptions): void {
     const defaults = {
       batching: {
-        maxBytes: Math.pow(1024, 2) * 5,
-        maxMessages: 1000,
-        maxMilliseconds: 100,
+        maxBytes: defaultOptions.publish.maxOutstandingBytes,
```

Comment:
If I'm reading this right - the idea was to drop the `* 5` here, right?

Reply:
That's the ultimate effect, yeah. There is a spreadsheet of the various default values in the libraries for different languages, and we're trying to normalize them. The original 5MB value was out of spec for that. I'll paste the link on chat in case that's interesting.